docs/AppFramework.md

```diff
@@ -196,21 +204,25 @@ spec:
 
 6. Apply the Custom Resource specification: `kubectl apply -f SearchHeadCluster.yaml`
 
-The App Framework detects the Splunk App archive files available in the App Source locations, and deploys the apps from the `adminApps`  folder to the deployer instance for local use. A Pod reset is triggered on the deployer to install any new or modified apps. The App Framework will also scan for changes to the App Source folders based on the polling interval, and deploy updated archives to the instance.
+The App Framework detects the Splunk app or add-on archive files available in the App Source locations, and deploys the apps from the `adminApps`  folder to the Deployer instance for local use. 
+
+The apps in the `searchApps` and `machineLearningApps` folders are deployed to the Deployer for use on the clustered search heads. The Deployer is responsible for deploying those apps to the search heads. The Splunk search head restarts are triggered by the contents of the Splunk apps deployed, and are not initiated by the App Framework.
```

Comment:
Similar to my previous comment for indexer cluster, maybe we can put this line , "The Splunk search head restarts are triggered by the contents of the Splunk apps deployed, and are not initiated by the App Framework.", as a Note?